app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -88,6 +94,13 @@ protected void onCreate(Bundle savedInstanceState) {
             }
         }
         WebViewProvider.preload(this);
+
+        SPEED_MODE = PreferenceManager.getDefaultSharedPreferences(this)
```

Comment:
You definitely would want to align these two (BLOCK_IMAGE and SPEED_MODE, with pref_key or without). So maybe keep them into Settings.java would make things easier.